src/m365/spo/commands/site/site-classic-set.ts

```diff
@@ -191,6 +193,30 @@ class SpoSiteClassicSetCommand extends SpoCommand {
             });
         });
       })
+      .then((): Promise<FormDigestInfo> => {
+        return this.ensureFormDigest(this.spoAdminUrl as string, logger, this.context, this.debug);
+      })
+      .then((res: FormDigestInfo): Promise<void> => {
+        this.context = res;
+        
+        if (!args.options.description) {
+          return Promise.resolve(undefined as any);
+        }
+
+        const requestOptions: any = {
+          url: `${args.options.url}/_api/web`,
+          headers: {
+            'IF-MATCH': '*',
+            'Accept': 'application/json;odata=verbose',
+            'content-type': 'application/json;odata=verbose',
```

Comment:
By removing `__metadata` from the payload, we can simplify the request and use `nometadata` to minimize the amount of data sent over the wire